src/test/series-utils.js

```diff
@@ -22,7 +22,6 @@ import test from 'tape';
 import 'babel-polyfill';
 
 import React from 'react';
-import equal from 'deep-equal';
 
 import {
```

Comment:
@chrisirhc , should be in a separate pull request. Much more changes are pending and blocked by this diff.